### PR TITLE
Modifying alerting logic based on alert classifications

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/node-agent#656

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n @nnDarshan please review

**Comment by GowthamShanmugam** [comment]
Tendrl/gluster-integration#449

**Comment by GowthamShanmugam** [comment]
@r0h4n more info added in Tendrl/node-agent#656

**Comment by GowthamShanmugam** [comment]
verified

File: tendrl/integrations/gluster/objects/definition/gluster.yaml [hunk 1]
```diff
@@ -13,6 +13,13 @@ namespace.integrations.gluster:
         volume_id:
           help: "Id of the volume"
           type: String
+      relationship:
+        utilization:
+          - volume_utilization
+        status:
+          - volume_status
+          - brick_status
+          - quorum
```
[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply tendrl/integrations/gluster/objects/definition/gluster.yaml]: @r0h4n

[the remark this answers is not in the document]
Comment (r0h4n) [reply tendrl/integrations/gluster/objects/definition/gluster.yaml]: @GowthamShanmugam send an email to gluster-dev and aravinda, shubhendu and atin requesting for the volume name to be added to those events

[the remark this answers is not in the document]
Comment (r0h4n) [reply tendrl/integrations/gluster/objects/definition/gluster.yaml]: Please include Gluster folks and @shtripat and get the volume name added to those events

[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply tendrl/integrations/gluster/objects/definition/gluster.yaml]: @r0h4n no need, subvolume gave volume name also, i am parsed and getting volume name from that

[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply tendrl/integrations/gluster/objects/definition/gluster.yaml]: https://github.com/Tendrl/gluster-integration/pull/453/files#diff-83685771786a9a125a3acb4fbca5215fR574

**Pull request merged**